internal/ethapi/backend.go

```diff
@@ -37,8 +37,19 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
-// Backend interface provides the common API services (that are provided by
-// both full and light clients) with access to necessary functions.
+type CeloBackend interface {
+	Backend
+	CeloBackendExtension
+}
+
+type CeloBackendExtension interface {
+	GetFeeBalance(ctx context.Context, atBlock common.Hash, account common.Address, feeCurrency *common.Address) (*big.Int, error)
+	GetExchangeRates(ctx context.Context, atBlock common.Hash) (common.ExchangeRates, error)
+	ConvertToCurrency(ctx context.Context, atBlock common.Hash, value *big.Int, feeCurrency *common.Address) (*big.Int, error)
+	ConvertToGold(ctx context.Context, atBlock common.Hash, value *big.Int, feeCurrency *common.Address) (*big.Int, error)
+}
```

Comment:
Any reason for not putting the function declarations directly into `CeloBackend`?

Reply:
No strong ones - those functions could be useful in different places than the API backend, so this would allow re-use of the interface. But as long as this is not used anywhere else it would be cleaner directly in the `CeloBackend`